SIT.Manager.Avalonia/ViewModels/PlayPageViewModel.cs

```diff
@@ -184,6 +177,9 @@ private async Task<string> LoginToServerAsync(Uri address)
                 CloseButtonText = _localizationService.TranslateSource("PlayPageViewModelButtonOk")
             }.ShowAsync();
         }
+        
+        QuickPlayText = _localizationService.TranslateSource("PlayPageViewModelQuickPlayText");
```

Comment:
Isn't this set elsewhere?

Reply:
> Isn't this set elsewhere?

I removed that line when I was making the server fix, but with change of statement in the aki service I added it back.

Reply:
Was a wrong line placement. Put it where it belongs.